### ISSUE TITLE
Substitutions Menu Not Working

### ISSUE THREAD

**Issue by CalebGrove** [nothingmagical/whiskey-issues#79]
I've notice the **Edit > Substitutions** menu doesn't seem to be working – while all substitutions are active in the app, none of the options are checked and click on them does nothing.

![](http://cl.ly/ZVeY/Screen%20Shot%202015-01-28%20at%2014.03.55.png)

**Comment by soffes** [comment]
Good call. I should probably remove this menu. I toggle these on and off programmatically frequently based on where you are focused. You can control them from Whiskey's preferences for now.

**Comment by CalebGrove** [comment]
*facepalm*
I don't know how I missed those options in the preferences, I've been trying to turn off smart dashes forever!

Side note: I don't want to start another issue for this as I'm not sure if it's even implementable (it's an edge case that may cause conflicts), but I'm using Kirby CMS for a project. The content files are simply Markdown, but with a special method of defining metadata:

    Title: My new blog post
    ----
    Date: 01/23/2015
    ----
    Description: This is a fancy new blog post.
    ----
    Content: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Isto modo, ne si avia quidem eius nata non esset. Gloriosa ostentatio in constituendo summo bono. Hic nihil fuit, quod quaereremus. Fortemne possumus dicere eundem illum Torquatum?

Each of the sections is separated with four dashes. Whisky interprets these as Setex-style headers (h2), which is correct in Markdown. However, in my case they are metadata, not headers. It would be nifty if Whisky either had some magical way of detecting this kind of metadata, but even an option to disable Setex-style headers would do the job.

However, that doesn't mean I think you should implement an option like this – mine is an edge case and my understanding is that you want Whisky to be fairly minimal without a huge number of preferences to set. Nor does it bother me that much. I just thought I'd put in on your radar to do with as you see fit. :)

**Comment by CalebGrove** [comment]
Actually, the Kirby metadata looks more like this, with a blank line between the dashes and the text:

    Title: My new blog post

    ----

    Date: 01/23/2015

    ----

    Description: This is a fancy new blog post.

    ----
    
    Content: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Isto modo, ne si avia quidem eius nata non esset.
